src/libcrun/linux.c

```diff
@@ -1863,7 +1863,10 @@ libcrun_set_terminal (libcrun_container_t *container, libcrun_error_t *err)
 
   ret = write_file ("/dev/console", NULL, 0, err);
   if (UNLIKELY (ret < 0))
-    return ret;
+    {
+      /* Ignore errors if the file creation fails.  */
+      crun_error_release (err);
```

Comment:
Should we check if the file exists? and only ignore those errors?

Reply:
sure, changed in the newer version